### PR TITLE
Update non-API docs

### PR DESCRIPTION
**Pull request by piskvorky**
Updating ancillary docs like `About`, `Intro`, `Install` etc.

### CODE REVIEW

File: docs/src/intro.rst [hunk 1]
```diff
@@ -9,14 +9,10 @@ topics from documents, as efficiently (computer-wise) and painlessly (human-wise
...
 Gensim is designed to process raw, unstructured digital texts ("*plain text*").
-The algorithms in `gensim`, such as **Latent Semantic Analysis**, **Latent Dirichlet Allocation** and **Random Projections**
-discover semantic structure of documents by examining statistical
-co-occurrence patterns of the words within a corpus of training documents.
-These algorithms are unsupervised, which means no human input is necessary -- you only need a corpus of plain text documents.
 
-Once these statistical patterns are found, any plain text documents can be succinctly
-expressed in the new, semantic representation and queried for topical similarity
-against other documents.
+The algorithms in Gensim, such as **Word2Vec**, **FastText**, **Latent Semantic Analysis**, **Latent Dirichlet Allocation** and **Random Projections**, discover semantic structure of documents by examining statistical co-occurrence patterns within a corpus of training documents. These algorithms are **unsupervised**, which means no human input is necessary -- you only need a corpus of plain text documents.
```
Comment (menshikh-iv) [inline docs/src/intro.rst]: please add links to documentation pages for each model, btw, who use RandomProjections? Maybe better to add something like TfIdfModel or Doc2Vec (but not RandomProjections)

File: docs/src/install.rst [hunk 2]
```diff
@@ -28,9 +28,6 @@ platform that supports Python 2.6+ and NumPy. Gensim depends on the following so
 * `NumPy <http://www.numpy.org>`_ >= 1.3. Tested with version 1.9.0, 1.7.1, 1.7.0, 1.6.2, 1.6.1rc2, 1.5.0rc1, 1.4.0, 1.3.0, 1.3.0rc2.
 * `SciPy <http://www.scipy.org>`_ >= 0.7. Tested with version 0.14.0, 0.12.0, 0.11.0, 0.10.1, 0.9.0, 0.8.0, 0.8.0b1, 0.7.1, 0.7.0.
 
-**Windows users** are well advised to try the `Enthought distribution <http://www.enthought.com/products/epd.php>`_,
-which conveniently includes Python & NumPy & SciPy in a single bundle, and is free for academic use.
-
 
 Install Python and `easy_install`
```
Comment (menshikh-iv) [inline docs/src/install.rst]: Please remove this section

File: docs/src/install.rst [hunk 3]
```diff
@@ -74,53 +70,51 @@ of computers, in :doc:`distributed`, you should install with::
...
 -----
 
 There are also alternative routes to install:
 
 1. If you have downloaded and unzipped the `tar.gz source <http://pypi.python.org/pypi/gensim>`_
-   for `gensim` (or you're installing `gensim` from `github <https://github.com/piskvorky/gensim/>`_),
+   for Gensim (or you're installing Gensim from `Github <https://github.com/piskvorky/gensim/>`_),
    you can run::
 
      python setup.py install
```
Comment (menshikh-iv) [inline docs/src/install.rst]: `pip install .`

File: docs/src/install.rst [hunk 4]
```diff
@@ -74,53 +70,51 @@ of computers, in :doc:`distributed`, you should install with::
...
 
 
-Testing `gensim`
-----------------
+Testing Gensim
+--------------
 
 To test the package, unzip the `tar.gz source <http://pypi.python.org/pypi/gensim>`_ and run::
 
     python setup.py test
```
Comment (menshikh-iv) [inline docs/src/install.rst]: `tox -e {py27,py35,py36}-{win,linux}`

File: docs/src/install.rst [hunk 5]
```diff
@@ -74,53 +70,51 @@ of computers, in :doc:`distributed`, you should install with::
...
 
-Gensim uses Travis CI for continuous integration: |Travis|_
+Gensim uses Travis CI for continuous integration, automatically running the full test suite on each pull request and commit: |Travis|_
```
Comment (menshikh-iv) [inline docs/src/install.rst]: In addition
- Appveyour (windows) and CircleCI (documentation)
- https://github.com/MacPython/gensim-wheels (python wheels)
- https://github.com/conda-forge/gensim-feedstock (conda packages)
- https://github.com/RaRe-Technologies/gensim-data (datasets & pre-trained models)

File: docs/src/install.rst [hunk 6]
```diff
@@ -74,53 +70,51 @@ of computers, in :doc:`distributed`, you should install with::
...
 
-.. |Travis| image:: https://api.travis-ci.org/piskvorky/gensim.png?branch=develop
-.. _Travis: https://travis-ci.org/piskvorky/gensim
+.. |Travis| image:: https://travis-ci.org/RaRe-Technologies/gensim.svg?branch=develop
+.. _Travis: https://travis-ci.org/RaRe-Technologies/gensim
```
Comment (menshikh-iv) [inline docs/src/install.rst]: Need to add (convert to from makdown first of course)
```
[![Conda-forge Build](https://anaconda.org/conda-forge/gensim/badges/version.svg)](https://anaconda.org/conda-forge/gensim)
[![Wheel](https://img.shields.io/pypi/wheel/gensim.svg)](https://pypi.python.org/pypi/gensim)
```

File: docs/src/install.rst [hunk 7]
```diff
@@ -9,11 +9,11 @@ Quick install
 
 Run in your terminal::
 
-  easy_install -U gensim
+  pip install --upgrade gensim
 
 or, alternatively::
 
-  pip install --upgrade gensim
+  easy_install -U gensim
```
[the remark this answers is not in the document]
Comment (piskvorky) [reply docs/src/install.rst]: OK.

File: docs/src/intro.rst (hunk 1 above)
Comment (piskvorky) [reply docs/src/intro.rst]: OK.

File: docs/src/install.rst (hunk 4 above)
Comment (piskvorky) [reply docs/src/install.rst]: What does this mean?

Comment (menshikh-iv) [reply docs/src/install.rst]: `tox -e <PYTHON_VERSION>-<OS_VERSION>`, for example, if I want to run tests on `Linux` with `python3.6`, I should run `tox -e py36-win`

File: docs/src/install.rst (hunk 6 above)
Comment (piskvorky) [reply docs/src/install.rst]: I don't think that's really needed or relevant here.

**Comment by menshikh-iv** [comment]
@piskvorky I added fixes for current files.
Please make last changes for `intro.rst`
- merge vector/sparse vector (I see no reason to have both)
- add BoW vector
- add more information about our corpus format (some object with `__iter__` that yield BoWs).

**Pull request merged**